web/packages/core/src/load-options.ts

```diff
@@ -336,6 +377,20 @@ export interface BaseLoadOptions {
      * @default true
      */
     polyfills?: boolean;
+
+    /**
+     * The handling mode of navigate_to_url website calls.
+     *
+     * @default OpenURLMode.Allow
+     */
+    openUrlMode?: OpenURLMode;
+
+    /**
+     * Whether and in what way the flash networking APIs can be accessed.
```

Comment:
Nit on grammar, sorry

Try just "Which flash networking APIs may be accessed"

Reply:
Yeah, that's better. :)